addons/sale/models/sale_order.py

```diff
@@ -507,6 +508,43 @@ def _compute_amounts(self):
             order.amount_tax = tax_totals['tax_amount_currency']
             order.amount_total = tax_totals['total_amount_currency']
 
+    def _add_base_lines_for_early_payment_discount(self):
+        """
+        When applying a payment term with an early payment discount, and when said payment term computes the tax on the
+        'mixed' setting, the tax computation is always based on the discounted amount untaxed.
+        Creates the necessary line for this behavior to be displayed.
+        :returns: array containing the necessary lines or empty array if the payment term isn't epd mixed
+        """
+        self.ensure_one()
+        epd_lines = []
+        if (
+                self.payment_term_id.early_discount
+                and self.payment_term_id.early_pay_discount_computation == 'mixed'
+                and self.payment_term_id.discount_percentage
+        ):
```

Comment:
```suggestion
        if (
            self.payment_term_id.early_discount
            and self.payment_term_id.early_pay_discount_computation == 'mixed'
            and self.payment_term_id.discount_percentage
        ):
```
I know this is because you are using PyCharm. Feel free to ignore it if you prefer a double tab

Reply:
I do prefer it 🙈

Reply:
Well, we want single tabs in sale, so please use a single tab :eyes:

Reply:
😢